test/test_issue833.py

```diff
@@ -0,0 +1,61 @@
+# test for https://github.com/RDFLib/rdflib/issues/833
+
+from rdflib import Graph, URIRef
+import unittest
+
+
+class TestIssue833(unittest.TestCase):
+
+    def setUp(self):
+        self.graph = Graph()
+
+        self.tarek = URIRef(u'tarek')
+        self.bob = URIRef(u'bob')
+        self.likes = URIRef(u'likes')
+        self.hates = URIRef(u'hates')
+        self.pizza = URIRef(u'pizza')
+        self.cheese = URIRef(u'cheese')
+
+        self.quads = []
+
+    def tearDown(self):
+        self.graph.close()
+
+    def prepareQuadsList(self):
+        tarek = self.tarek
+        bob = self.bob
+        likes = self.likes
+        hates = self.hates
+        pizza = self.pizza
+        cheese = self.cheese
+
+        self.quads.append((tarek, likes, pizza, self.graph))
+        self.quads.append((tarek, likes, cheese, self.graph))
+        self.quads.append((tarek, likes, bob, self.graph))
+        self.quads.append((bob, likes, cheese, self.graph))
+        self.quads.append((bob, hates, pizza, self.graph))
+
+    def testAddN(self):
+        asserte = self.assertEqual
```

Comment:
you don't really need to define `asserte`. It's defined in other places because it's used so many times in the same method.